tests/functions/test_origin_request.py

```diff
@@ -70,7 +71,17 @@ def test_origin_request(
 
     event = {"Records": [{"cf": {"request": {"uri": req_uri, "headers": {}}}}]}
 
-    request = OriginRequest(conf_file=CONF_PATH).handler(event, context=None)
+    request_obj = OriginRequest(conf_file=CONF_PATH)
+    # request.conf["logging"] = logger_conf
+    request_obj.conf["logging"]["formatters"]["default"][
```

Comment:
This is weird, you're adjusting the config after the object is created.

I guess this is relying on the behavior that the logger isn't configured until the first time it's used, and it's not used within the constructor?  This is setting a bit of a trap, e.g. adding a logging statement in \_\_init\_\_ seems like an innocent thing to do, but if a developer tries that then they'll cause this test to start failing.  Ideally the test should not be so brittle.

Can we just feed the desired config into the object when constructing it instead?  Why do we feed one "wrong" config, and then modify it later, relying on the implementation detail that the config hasn't been accessed yet?